packages/desktop-client/src/components/SidebarWithData.js

```diff
@@ -1,23 +1,58 @@
 import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
-import { withRouter } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { withRouter, useHistory } from 'react-router';
 
 import { bindActionCreators } from 'redux';
 
 import * as actions from 'loot-core/src/client/actions';
+import { closeBudget } from 'loot-core/src/client/actions/budgets';
 import * as queries from 'loot-core/src/client/queries';
 import { send } from 'loot-core/src/platform/client/fetch';
 import {
   Button,
   Input,
   InitialFocus,
-  Text
+  Text,
+  Tooltip,
+  Menu
 } from 'loot-design/src/components/common';
 import { Sidebar } from 'loot-design/src/components/sidebar';
 import { styles, colors } from 'loot-design/src/style';
+import ExpandArrow from 'loot-design/src/svg/ExpandArrow';
 
 function EditableBudgetName({ prefs, savePrefs }) {
+  let dispatch = useDispatch();
+  let history = useHistory();
   const [editing, setEditing] = useState(false);
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  function onMenuSelect(type) {
+    setMenuOpen(false);
+
+    switch (type) {
+      case 'rename':
+        setEditing(true);
+        break;
+      case 'settings':
+        history.push('/settings');
+        break;
+      case 'help':
+        window.open('https://actualbudget.github.io/docs', '_blank');
+        break;
+      case 'close':
+        dispatch(closeBudget());
+        break;
+      default:
+    }
+  }
+
+  let items = [
+    { name: 'rename', text: 'Rename Budget' },
+    { name: 'settings', text: 'Settings' },
+    { name: 'help', text: 'Help' },
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the `help` link should either be hidden when using the electron app, or it should be made to work (currently it won't work).

To check if the app is electron you can use `!Platform.isBrowser`